### ISSUE TITLE
initialize error in 8.1.20,8.1.19 is ok

### ISSUE THREAD

**Issue by Dreampie** [eclipse/jetty.project#908]
```
16-09-07 17:08:14.239 [qtp82681671-16 Selector0] WARN  org.eclipse.jetty.io.nio - 
java.lang.NoClassDefFoundError: Could not initialize class org.eclipse.jetty.server.AsyncContinuation
	at org.eclipse.jetty.server.Request.<init>(Request.java:182) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AbstractHttpConnection.<init>(AbstractHttpConnection.java:162) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AsyncHttpConnection.<init>(AsyncHttpConnection.java:50) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newConnection(SelectChannelConnector.java:285) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newConnection(SelectChannelConnector.java:325) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newEndPoint(SelectChannelConnector.java:272) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newEndPoint(SelectChannelConnector.java:331) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.createEndPoint(SelectorManager.java:842) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:496) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290) [jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65]
16-09-07 17:08:14.239 [qtp82681671-22 Selector1] WARN  org.eclipse.jetty.io.nio - 
java.lang.NoSuchMethodError: java.lang.Error.<init>(Ljava/lang/String;Ljava/lang/Throwable;ZZ)V
	at org.eclipse.jetty.continuation.ContinuationThrowable.<init>(ContinuationThrowable.java:49) ~[jetty-continuation-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AsyncContinuation.<clinit>(AsyncContinuation.java:57) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.Request.<init>(Request.java:182) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AbstractHttpConnection.<init>(AbstractHttpConnection.java:162) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AsyncHttpConnection.<init>(AsyncHttpConnection.java:50) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newConnection(SelectChannelConnector.java:285) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newConnection(SelectChannelConnector.java:325) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newEndPoint(SelectChannelConnector.java:272) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newEndPoint(SelectChannelConnector.java:331) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.createEndPoint(SelectorManager.java:842) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:496) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290) [jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65]
16-09-07 17:08:14.312 [qtp82681671-16 Selector0] WARN  org.eclipse.jetty.io.nio - 
java.lang.NoClassDefFoundError: Could not initialize class org.eclipse.jetty.server.AsyncContinuation
	at org.eclipse.jetty.server.Request.<init>(Request.java:182) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AbstractHttpConnection.<init>(AbstractHttpConnection.java:162) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AsyncHttpConnection.<init>(AsyncHttpConnection.java:50) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newConnection(SelectChannelConnector.java:285) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newConnection(SelectChannelConnector.java:325) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newEndPoint(SelectChannelConnector.java:272) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newEndPoint(SelectChannelConnector.java:331) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.createEndPoint(SelectorManager.java:842) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:496) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290) [jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65]
16-09-07 17:08:14.364 [qtp82681671-22 Selector1] WARN  org.eclipse.jetty.io.nio - 
java.lang.NoClassDefFoundError: Could not initialize class org.eclipse.jetty.server.AsyncContinuation
	at org.eclipse.jetty.server.Request.<init>(Request.java:182) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AbstractHttpConnection.<init>(AbstractHttpConnection.java:162) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AsyncHttpConnection.<init>(AsyncHttpConnection.java:50) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newConnection(SelectChannelConnector.java:285) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newConnection(SelectChannelConnector.java:325) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newEndPoint(SelectChannelConnector.java:272) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newEndPoint(SelectChannelConnector.java:331) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.createEndPoint(SelectorManager.java:842) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:496) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290) [jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65]
16-09-07 17:08:19.380 [qtp82681671-16 Selector0] WARN  org.eclipse.jetty.io.nio - 
java.lang.NoClassDefFoundError: Could not initialize class org.eclipse.jetty.server.AsyncContinuation
	at org.eclipse.jetty.server.Request.<init>(Request.java:182) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AbstractHttpConnection.<init>(AbstractHttpConnection.java:162) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AsyncHttpConnection.<init>(AsyncHttpConnection.java:50) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newConnection(SelectChannelConnector.java:285) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newConnection(SelectChannelConnector.java:325) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newEndPoint(SelectChannelConnector.java:272) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newEndPoint(SelectChannelConnector.java:331) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.createEndPoint(SelectorManager.java:842) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:496) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290) [jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65]
16-09-07 17:08:49.397 [qtp82681671-22 Selector1] WARN  org.eclipse.jetty.io.nio - 
java.lang.NoClassDefFoundError: Could not initialize class org.eclipse.jetty.server.AsyncContinuation
	at org.eclipse.jetty.server.Request.<init>(Request.java:182) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AbstractHttpConnection.<init>(AbstractHttpConnection.java:162) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.AsyncHttpConnection.<init>(AsyncHttpConnection.java:50) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newConnection(SelectChannelConnector.java:285) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newConnection(SelectChannelConnector.java:325) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector.newEndPoint(SelectChannelConnector.java:272) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.server.nio.SelectChannelConnector$ConnectorSelectorManager.newEndPoint(SelectChannelConnector.java:331) ~[jetty-server-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.createEndPoint(SelectorManager.java:842) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$SelectSet.doSelect(SelectorManager.java:496) ~[jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.io.nio.SelectorManager$1.run(SelectorManager.java:290) [jetty-io-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:608) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at org.eclipse.jetty.util.thread.QueuedThreadPool$3.run(QueuedThreadPool.java:543) [jetty-util-8.1.20.v20160902.jar:8.1.20.v20160902]
	at java.lang.Thread.run(Thread.java:695) [na:1.6.0_65]
```

**Comment by jmcc0nn3ll** [comment]
You are using Java 7 correct?

**Comment by Dreampie** [comment]
java 6

**Comment by jmcc0nn3ll** [comment]
ok, don't use 8.1.20

That change was added in to address a memory issue being expressed via osgi on app restart and was the only change in the release.  

Looks like we'll have to mull over a way around that in java 6.

**Comment by joakime** [comment]
Jetty 8 (which is EOL btw), does not support Java 6 (which is also EOL).
It has a Java 7 (hey, even this is EOL) minimum requirement.

The last version of Jetty 8 that was Java 6 compatible was 8.1.9.v20130131

**Comment by Dreampie** [comment]
ok, thanks

**Comment by jmcc0nn3ll** [comment]
@Dreampie I reverted that change and released 8.1.21 so you can keep tracking the latest 8.1.x releases with Java 6.  I also released the needed Java 1.7 change under Jetty 8.2.0 which now has a Java 7 minimum version.
